README.md

```diff
@@ -27,35 +78,57 @@ FetchContent_Declare(mtest
 FetchContent_MakeAvailable(mtest)
 ```
 
-Now the CMake library target `mtest` is available. For [example](example/CMakeLists.txt):
-```cmake
-add_executable(my_example_test my_test.c)
-target_link_libraries(my_example_test mtest)
+Updates can be done by changing the `URL` and `URL_HASH` in the first approach,
+or by changing the `GIT_TAG` in the second approach.
 
-# Automatically discover test cases in my_example_test and add them to CTest.
-discover_tests(my_example_test)
+### Using Git external submodules
+
+Lets you manage mtest as a linked directory managed by git. This is the
+more traditional way of linking dependencies.
+
+Open a terminal and ensure you are inside your project's root
+(where your main CMakeLists.txt file is located). Then type the following:
+```bash
+mkdir extern
+git submodule add https://github.com/MortenSchou/mtest.git extern/mtest
 ```
 
-## Usage
-Example test file [`my_test.c`](example/my_test.c):
-```C
-#include "mtest.h"
+Now you can add the following to your CMakeLists.txt:
+```cmake
+if(NOT EXISTS "${PROJECT_SOURCE_DIR}/extern/mtest/CMakeLists.txt")
+    message(FATAL_ERROR "The git submodules were not downloaded! GIT_SUBMODULE was turned off or failed. Did you forget to run `git submodule init` after cloning?")
+else()
+    add_subdirectory(extern/mtest)
+endif()
+```
 
-TEST_CASE(my_first_test_case, {
-  int i = 42;
-  CHECK_EQ_INT(1, i);    // Test fails, but we continue running.
-  REQUIRE_GT_INT(i, 0);  // This test is executed and succeeds, but the test case failed due to the previous line.
-})
+Updating mtest can be done with the `git submodule update` command, then
+using git to commit the updated submodule directory.
 
-TEST_CASE(my_other_test_case, {
-  double i = 3.14;
-  // When comparing doubles, we need to specify a tolerance - here we choose 0.01
-  REQUIRE_EQ_DOUBLE(4.0, i, 0.01);  // Test fails and the test case exits.
-  CHECK_TRUE(1);                    // Not executed but it would succeed.
-})
+### NOT RECOMMENDED: Using system installed `mtest`
 
-// This macro creates a main function that can call each test case, and it tells CTest which test cases are available. 
-MAIN_RUN_TESTS(my_first_test_case, my_other_test_case)
+This is likely not what you want, and it is recommended you pick one of the
+other approaches, as this does not ensure that anyone using your project will
+have the same mtest library installed as you, or even that its installed.
+
+However, this approach can be helpful if you are packaging your project, or have
+otherwise installed mtest as part of your operating system.
+
+If you do not have mtest already installed, you can run the following two
+commands:
+```bash
+cmake -B cmake_build -S . -DCMAKE_BUILD_TYPE=Release -DMTEST_INSTALL=ON
```

Comment:
Looks like the MTEST_INSTALL option is no longer used.
```suggestion
cmake -B cmake_build -S . -DCMAKE_BUILD_TYPE=Release
```

Reply:
Oops! Its now fixed.